documentation/src/docbkx/chapter22-improve.xml

```diff
@@ -418,6 +418,200 @@
 
 	</section>
 
+	<section id="merge_duplicates">
+
+		<title>
+			Merge duplicates
+			<inlinemediaobject>
+				<imageobject>
+					<imagedata fileref="notice_commercial_editions_only.png"
+						format="PNG" />
+				</imageobject>
+			</inlinemediaobject>
+		</title>
+
+		<para>Merging duplicates is the next step after detecting them in a
+			dataset. It helps to restore a single version of truth by combining
+			information from all the duplicate records representing the same
+			physical entity.
+		</para>
+
+		<para>
+			In this section, we assume that the steps from previous section
+			"Duplicate detection" are completed and the duplicates are available
+			in a staging table "Duplicates export".
```

Comment:
instead of "staging table" I think "staging datastore" or just "datastore" is more correct.